src/os/bluestore/BitAllocator.cc

```diff
@@ -123,8 +123,7 @@ void BmapEntry::_init_bit_mask()
   BmapEntry::m_bit_mask_init = true;
 }
 
-BmapEntry::BmapEntry(CephContext* cct, bool full)
-  : cct(cct)
+BmapEntry::BmapEntry(CephContext*, const bool full)
```

Comment:
You're right. I deferred the clean-up to not introduce dependencies between patches as I'm going to touch `BitMapZone` as well. Having `CephContext*` there  costs us additional 2 MiB on 1 TiB SSD. :-(